Package.swift

```diff
@@ -15,14 +15,15 @@ let package = Package(
     name: "SpeziOnboarding",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v17)
+        .iOS(.v17),
```

Comment:
Did you test it on the iPad as well? Any layout issues on the wider screen?

Reply:
Looks fine on iPadOS. But layout is really out of scope for SpeziOnboarding. If you are using SpeziOnboarding in sheets your are fine and otherwise you will need to make sure to properly frame the onboarding views according to your needs.

Reply:
Makes sense, thx!